### DISCUSSION TITLE
Unique fields are counting itself as duplicate when editing

### DISCUSSION THREAD

**Discussion by Xoshbin** [filamentphp/filament#2280] [Help] [answered]
### Package

filament/filament

### Package Version

v2.10.44

### Laravel Version

v8.0

### PHP Version

PHP 8.0

### Bug description

I have a form with a "slug" field that should be "unique()", but the issue is I have to change that field with increase or decrease at least a character to be able to save the form because it's counting the field as duplicate, I have to do my changes in the form then edit the unique field to be able to save it, then I have to come back to the form to change the slug back to it's original and save it again.

**Comment by ryangjchandler** [comment, marked as answer]
@Xoshbin Are you ignoring the current record? `->unique(ignoreRecord: true)`

**Reply by aurawindsurfing** [reply]
@ryangjchandler maybe should be added as a default to unique method as this is what it is suppose to do

**Comment by Xoshbin** [comment]
No I'm not using that

**Comment by ryangjchandler** [comment]
@Xoshbin Add that to the field and it should work.

**Reply by Xoshbin** [reply]
Thank you, it worked 🤦‍♂️
